Terrain/border.py

```diff
@@ -1,7 +1,7 @@
 import copy
 
 import Terrain.terraingen as terraingen
-from Terrain.timers import FunctionTimer
+from Terrain.timer import FunctionTimer
```

Comment:
timing is good

Reply:
timing is a good name yes, i will change